htmd/molecule/voxeldescriptors.py

```diff
@@ -259,9 +259,9 @@ def _getRadii(mol):
             raise ValueError('PDBQT element {} does not exist in mappings.'.format(elem))
         elem = mappings[elem]
 
-        if elem in VDW.elements:
-            rad = radiusByElement(elem)
-        else:
+        try:
```

Comment:
no, thats just ugly. why try catch when you can check?

Reply:
Or check in the function instead of here

Reply:
sorry, stupidity from my side :)